moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp

```diff
@@ -456,6 +511,28 @@ ompl_interface::ModelBasedPlanningContextPtr ompl_interface::PlanningContextMana
     }
   }
 
+  // State space selection process
```

Comment:
This is definitely a great start, but this should go into the tutorial as well (maybe even above `longest_valid_segment_fraction` because many readers will be running once they get there :wink:  ).

Reply:
Yes I will add this to the tutorial. I also added it to the readme in the demo added here.